### DISCUSSION TITLE
What is "psapi"?

### DISCUSSION THREAD

**Discussion by Patterner** [ThioJoe/jpeg2png#1] [Q&A]
"mold: fatal: library not found: psapi"  which is correct. unfortunately there is only a libpsai in wine-dev which is not installed.

trying to link without -lpsapi works, so it seems to be unnecessary.

where would i find the library?

**Comment by jwillikers** [comment]
It's a Windows only library, [Process Status API](https://learn.microsoft.com/en-us/windows/win32/psapi/process-status-helper). The `Makefile` erroneously adds the dependency whenever the compiler is `clang`.
